packages/atomic/src/components/facets-v1/facet-value-box/facet-value-box.tsx

```diff
@@ -12,8 +12,9 @@ export const FacetValueBox: FunctionalComponent<FacetValueProps> = (props) => {
     <li key={props.displayValue}>
       <button
         part="value-box"
+        onMouseUp={(e) => (e.target as HTMLButtonElement).blur()}
```

Comment:
I'm not sure I understand, why are we blurring this button after it's pressed? This could potentially cause issues on mobile devices running accessibility tools.

If possible, instead of blurring the pressed button, I think we should focus a different (relevant) element.

Reply:
What kind of issues? It's a blur when using the mouse, so the focus state doesn't stay active when you click on the facet. Using the keyboard, you'll keep the focus were you previously were.

Reply:
Is this for a CSS fix only ? If so, can this be fixed with CSS ? 

IIRC, there's a way to distinguish the focused state with keyboard and focused state with mouse.

Reply:
It's called focus-visible, it's not fully supported, doesn't work inside Safari. We can default to focus for the browsers that don't support it OR we can use polyfills + perhaps a custom stencil plugin.

**Workaround:**
 https://css-tricks.com/platform-news-using-focus-visible-bbcs-new-typeface-declarative-shadow-doms-a11y-and-placeholders/

**Polyfills**:
https://github.com/csstools/postcss-focus-visible
https://github.com/WICG/focus-visible

I'll merge this for now, but open a JIRA to explore these options